### ISSUE TITLE
TypeError instantiating client interceptor

### ISSUE THREAD

**Issue by raybotha** [opentracing-contrib/python-grpc#8]
My server-side tracing is working well, however testing a client-side interceptor fails immediately with this error:
```
Traceback (most recent call last):
  File "client.py", line 29, in <module>
    channel = intercept_channel(channel, interceptor)
  File "/home/ray/.virtualenvs/grpc-GRTZPqPV/lib/python3.6/site-packages/grpc_opentracing/grpcext/__init__.py", line 95, in intercept_channel
    return _interceptor.intercept_channel(channel, *interceptors)
  File "/home/ray/.virtualenvs/grpc-GRTZPqPV/lib/python3.6/site-packages/grpc_opentracing/grpcext/_interceptor.py", line 216, in intercept_channel
    result = _InterceptorChannel(result, interceptor)
TypeError: Can't instantiate abstract class _InterceptorChannel with abstract methods close
```

This is the code:
```
import grpc
import yaml
from jaeger_client import Config
from grpc_opentracing import open_tracing_client_interceptor
from grpc_opentracing.grpcext import intercept_channel
from calculator import CalculatorStub

SERVICE = "calculator-client"

with open("config.yaml") as file:
    config = yaml.load(file)


def init_tracer(service_name=SERVICE):
    tracer_config = Config(config=config["jaeger"],
                           service_name=service_name,
                           validate=True)
    tracer = tracer_config.initialize_tracer()
    return tracer


tracer = init_tracer()
interceptor = open_tracing_client_interceptor(tracer, log_payloads=True)
channel = grpc.insecure_channel("localhost:50051")
channel = intercept_channel(channel, interceptor)
client = CalculatorStub(channel)
```

This is my config:
```
jaeger:
  sampler:
    type: const
    param: 1
  local_agent:
    reporting_host: 127.0.0.1
    reporting_port: 5775
  logging: true
```
```
Python 3.6.5, opentracing==1.3.0, grpcio-opentracing==1.1.1
```
Thanks

**Comment by raybotha** [comment]
Looks like https://github.com/opentracing-contrib/python-grpc/pull/6 fixed this, but 1.1.12 is not on PyPi. Would you be able to push it @rnburn? :)

**Comment by rnburn** [comment]
Done
